### DISCUSSION TITLE
Cubefs 后端存储接入蓝光存储

### DISCUSSION THREAD

**Discussion by fengjiannan2010** [cubefs/cubefs#3390] [Ideas]
我想使用cubefs 开发蓝光存储系统，单张光盘容量为100GB,一个蓝光库体可以容纳20000张光盘，整体容量为2PB,基于cubefs 对外提供S3,POSIX,HDFS。我希望在我的前置机提供一个高速的缓存空间，并把数据按照单张光盘的容量进行切分，并保持文件的完整性（文件不切分，或者动态合并），写入蓝光设备，并更新元数据信息标识文件存储在蓝光介质。能给提供下具体的实现方法吗？在那一层入手进行适配。也就是说DataNode 存储的数据可以指定存储介质，热数据存储在SDD,温数据存储在HDD,冷数据存储在蓝光设备。

**Comment by leonrayang** [comment]
可以，需要等待混合云版本上线，要二次开发，欢迎为微信群内联系我们
